src/app/work-item/work-item-list/work-item-list.component.ts

```diff
@@ -48,31 +50,41 @@ export class WorkItemListComponent implements OnInit {
 
   ngOnInit(): void {
     this.listenToEvents();
-    this.reloadWorkItems();
     this.loggedIn = this.auth.isLoggedIn();
   }
 
   // model handlers
 
-  reloadWorkItems(): void {
+  initWiItems(event: any): void {
+    this.loadWorkItems(event.pageSize);
+  }
+
+  loadWorkItems(pageSize: number = 20): void {
```

Comment:
pageSize is defined as 10 in infinitescroll.directive. Why is it reset to 20 here?